src/OpenApi/test/Microsoft.AspNetCore.OpenApi.Tests/Integration/snapshots/OpenApiDocumentIntegrationTests.VerifyOpenApiDocument_documentName=v2.verified.txt

```diff
@@ -28,12 +28,12 @@
                     "type": "string",
                     "externalDocs": {
                       "description": "Documentation for this OpenAPI schema",
-                      "url": "https://example.com/api/docs/schemas/string"
+                      "url": "https://example.com/api/docs/schemas/String"
```

Comment:
This change looks odd. I suppose the URL is made up so there is no right or wrong -- but usually URLs follow conventions, like "all lowercase" (as the previous URL did) or "camel cased segments". Here there is a mix of lowercase and Pascal case, which is ... odd.

Reply:
Yep, this is result of the same reason in https://github.com/dotnet/aspnetcore/pull/59480#discussion_r1913495903. Because we use `JsonSchemaType.String.ToString()` the default casing produced is titlecase. We can fix this by just using the literal "string" directly.